plugin/mutate/source/dextool/plugin/mutate/backend/database/type.d

```diff
@@ -225,3 +225,17 @@ struct MutantMetaData {
         attr.match!((NoMetadata a) {}, (NoMut a) => put(w, "nomut"));
     }
 }
+
+struct Rationale {
+  string time;
+  string rationale;
+}
+
+// Mutant manually marked in db
+struct MarkedMutant {
+    MutationId mutant_id;
+    Path path;
```

Comment:
My thought was that the information is then used when the analyze-phase is rerun in order to tell the user which marked mutants will become obsolete (not yet implemented).

Or are you meaning that I could use AbsolutePath, instead of Path?

Reply:
Path is good. I just didn't understand that it's was too be used as information. Path, line, column and annotation should be pretty good for the user

Reply:
Should we maybe add mutant kind?

Reply:
I am currently implementing this, and I have the following:
`
| File | Line | Column | Status | Rationale |
`

Is this really enough for the user, or is something missing?

Reply:
Yes, mutant kind is probably a good idea. file+line+column+kind makes it globally unique